disco_aws_automation/disco_deploy.py

```diff
@@ -695,7 +704,8 @@ def update(self, dry_run=False, deployment_strategy=None, ticket_id=None):
                 raise
 
         socify_helper.send_event(status=status,
-                                 hostclass=(DiscoBake.ami_hostclass(ami) if ami else None),
+                                 hostclass=(hostclass if ami else None),
```

Comment:
When no ami is defined and a ticket was specify the code also send an event to Socify to add a comment to indicate that the deploy failed. So the test is needed.
If we don't need to update the Ticket in that case we can remove the call to Socify.